include/libsleigh.hh

```diff
@@ -33,3 +34,5 @@
 #include "types.h"
 #include "xml.hh"
 #pragma GCC diagnostic pop
+
+#include "Support.h"
```

Comment:
This feels odd to include here because the support header is not actually a part of NSA's Ghidra/Sleigh. I feel like it would make more sense to keep the support header separate from this file.

Also, I think this causes issues if someone only links against `sleigh::sla`, since the `sleigh::support` target isn't a `PUBLIC` link in the CMake, so the `libsupport.a` will never be linked and will fail to compile.

Reply:
Isn't that the case with not linking against `sleigh::decomp` too? It's just a convenience header so I figured it'd make sense to just include the helpers that you'd typically use alongside the library.

Reply:
I am in agreement with @tetsuo-cpp.

Reply:
Fair enough 👍